integration/auction_test.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+# Copyright Intel Corp. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+SCRIPTDIR="$(dirname $(readlink --canonicalize ${BASH_SOURCE}))"
+FPC_TOP_DIR="${SCRIPTDIR}/.."
+CONFIG_HOME="${SCRIPTDIR}/config"
+
+. ${SCRIPTDIR}/common_utils.sh
+. ${SCRIPTDIR}/common_ledger.sh
+
+CC_ID=ecc_auction_test
```

Comment:
It is failing for me and this should be the problem: 
you have updated the id but not vscc;
there is also an mrenclave not found -- I am still debugging

Reply:
> you have updated the id but not vscc;
i only changed id to make docker images distinct if you run multiple setups, e.g., integration test and example, on same machine. In any case, not sure what id you mean when you refer to vscc?  This is a reference to code (and not an instance of the chaincode)?

> there is also an mrenclave not found -- I am still debugging
not sure what you are referring to here